app/Http/Kernel.php

```diff
@@ -37,6 +37,7 @@ class Kernel extends HttpKernel
             \Illuminate\View\Middleware\ShareErrorsFromSession::class,
             \App\Http\Middleware\VerifyCsrfToken::class,
             \Illuminate\Routing\Middleware\SubstituteBindings::class,
+            \App\Http\Middleware\LocalizationMiddleware::class
```

Comment:
array nhiều dòng, dòng cuối phải có ","
```suggestion
            \App\Http\Middleware\LocalizationMiddleware::class,
```